website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+1. Users with a session longer than a certain amount of time
+1. Users that interacted with a particular feature
+
+From there, the Analyst will build out the initial documentation and write down what they want the final dataset to look like. If your Analyst is not trained as an Analytics Engineer, this is the point that they will need to hand the project over to a Data Engineer to build the [model](https://docs.getdbt.com/docs/building-a-dbt-project/building-models).
+
+The first time that a team of Analysts and Data Engineers build a curated dataset, they will often expect this process to be a straight line to completion. Expectations are that the process will look something like this:
+
+![The expectation](/img/blog/2022-09-28-analyst-to-ae/first_loop.png)
+
+What quickly becomes apparent is that this push for a linear process often ends up creating three unanticipated “loops”:
+
+![The reality](/img/blog/2022-09-28-analyst-to-ae/multiple_loops.png)
+
+## Loop 1: Reacting to findings in the raw data
+
+Let’s say that your Analyst has defined the business need as an `Active Users` dataset with “Unique users that login during a given day”. The Analyst will try to do as much discovery work up-front as possible, because it’s hard to predict exactly what you’ll find in the raw data. When a Data Engineer gets stuck writing a model, they’ll need to reach out to the Analyst for some additional discovery. When an Analyst-turned-Analytics-Engineer has a question while writing a model, they don’t need to wait to talk to anyone, and they can begin researching it immediately. Which brings us to our first point:
+
+<h3 align="center">Analysts already know the data that they want.</h3>
+
+If the raw `Login` dataset contains two different date (`Login_Date` and `Session_Date`) fields, a Data Engineer would be stuck. They can’t just guess because using the wrong date field will create an entirely different metric! So they must go back to the Analyst for clarity on which field to use. We’ve just had a full loop with two hand-offs, and the Data Engineer hasn’t even begun building the model yet.
+
+| Login_Date | Session_Date | User_Id |
+|------------|--------------|---------|
+| 2022-08-01 | 2022-08-01   | 123     |
+| 2022-08-01 | 2022-08-03   | 123     |
+| 2022-08-01 | 2022-08-04   | 123     |
+| 2022-08-01 | 2022-08-01   | 427     |
+| 2022-08-02 | 2022-08-02   | 864     |
+| 2022-08-03 | 2022-08-03   | 864     |
+| 2022-08-04 | 2022-08-04   | 975     |
+| 2022-08-04 | 2022-08-04   | NULL    |
+
+In comparison, your new Analytics Engineer could quickly pivot based on the findings in the raw data. They can skip the endless loops of “discovery”, because they can react in-the-moment. When they see the two date fields for `Login_Date` and `Session_Date`, they can instantly apply their product knowledge, compare against examples, or reach out to their business contacts to diagnose the difference and select the correct date for the model.
```

Comment:
```suggestion
In comparison, your new Analytics Engineer could quickly pivot based on the findings in the raw data. They can skip the endless loops of “discovery”, because they can react in the moment. When they see the two date fields for `Login_Date` and `Session_Date`, they can instantly apply their product knowledge, compare against examples, or reach out to their business contacts to diagnose the difference and select the correct date for the model.
```